diff --git a/browser/base/content/aboutDialog.js b/browser/base/content/aboutDialog.js
--- a/browser/base/content/aboutDialog.js
+++ b/browser/base/content/aboutDialog.js
@@ -52,7 +52,7 @@ function init() {
   ]);
   let versionIdKey = "base";
   let versionAttributes = {
-    version: AppConstants.MOZ_APP_VERSION_DISPLAY,
+    version: `${AppConstants.MOZ_APP_VERSION_DISPLAY} (Firefox ${AppConstants.ZEN_FIREFOX_VERSION})`,
   };
 
   let arch = Services.sysinfo.get("arch");
@@ -64,7 +64,7 @@ function init() {
   }
 
   let version = Services.appinfo.version;
-  if (/a\d+$/.test(version)) {
+  if (version.includes("t")) {
     versionIdKey += "-nightly";
     let buildID = Services.appinfo.appBuildID;
     let year = buildID.slice(0, 4);
@@ -85,21 +85,6 @@ function init() {
     versionAttributes
   );
 
-  // Show a release notes link if we have a URL.
-  let relNotesLink = document.getElementById("releasenotes");
-  let relNotesPrefType = Services.prefs.getPrefType(
-    "app.releaseNotesURL.aboutDialog"
-  );
-  if (relNotesPrefType != Services.prefs.PREF_INVALID) {
-    let relNotesURL = Services.urlFormatter.formatURLPref(
-      "app.releaseNotesURL.aboutDialog"
-    );
-    if (relNotesURL != "about:blank") {
-      relNotesLink.href = relNotesURL;
-      relNotesLink.hidden = false;
-    }
-  }
-
   if (AppConstants.MOZ_UPDATER) {
     gAppUpdater = new appUpdater({ buttonAutoFocus: true });
 
@@ -125,14 +110,6 @@ function init() {
       window.close();
     });
   if (AppConstants.MOZ_UPDATER) {
-    document
-      .getElementById("aboutDialogHelpLink")
-      .addEventListener("click", () => {
-        openHelpLink("firefox-help");
-      });
-    document
-      .getElementById("submit-feedback")
-      .addEventListener("click", openFeedbackPage);
     document
       .getElementById("checkForUpdatesButton")
       .addEventListener("command", () => {
